### ISSUE TITLE
V4 Quickstart hard to follow

### ISSUE THREAD

**Issue by dennistruemper** [lustre-labs/lustre#62]
Hi, I just tried to follow https://hexdocs.pm/lustre/4.0.0-rc1/guide/01-quickstart.html and I did not look in the URL. So I followed the instructions and ended up with a Version 3 install. Took me as a gleam/lustre beginner some time to figure out what is wrong. 

Then I took a look at some examples and in there I found: 

`Note: this guide is written for Lustre v4. The latest stable release of Lustre is v3. To follow along with this guide, you need to manually edit your gleam.toml and change the required version of lustre to "4.0.0-rc.2".`

So I'd suggest to add some info in the quickstart to check for Version 4 to make it easier to follow

**Comment by hayleigh-dot-dev** [comment]
Hey there thanks for the feedback (: Louis released Gleam v1 and then the subsequent prime/theo videos all happened while I've been away travelling for work so I haven't had much time at all to keep on top of things. V4 should be out for real fairly soon ^.^
